drivers/storage/portworx/component/csi.go

```diff
@@ -513,6 +523,24 @@ func (c *csi) createDeployment(
 	return nil
 }
 
+func (c *csi) findPreinstalledCSISnapshotController() error {
+	podList := &v1.PodList{}
+	err := c.k8sClient.List(context.TODO(), podList)
```

Comment:
this would list the default namespace I think. @ggriffiths shall we search all namespaces?

Reply:
I would search in all namespaces only once as that is expensive

Reply:
from unit test this already list pods from all namespaces, and we only list once in the first reconcile loop.

Reply:
Sounds good :+1: